tests/e2e/go.mod

```diff
@@ -9,4 +9,4 @@ require (
 	gopkg.in/yaml.v2 v2.3.0
 	k8s.io/klog/v2 v2.4.0
 	sigs.k8s.io/kubetest2 v0.0.0-20210115020551-4275dd0a0d63
-)
```

Comment:
@srikiz I think these trailing newline changes in go.mod and go.sum need to be reverted as well.

Reply:
Sure, updated.